### ISSUE TITLE
roo2matplotlib creates only black and white print out

### ISSUE THREAD

**Issue by oozaru-hunter** [rootpy/rootpy#628]
I'm trying to read a histogram from a ROOT file and to plot it with matplotlib (both as histogram and errobar).

Following code is used:

````#!/usr/bin/python
import matplotlib
matplotlib.use('Agg') # do this before importing pyplot or root2matplotlib
from rootpy import ROOT 
import rootpy.plotting.root2matplotlib as rplt
from rootpy.io import root_open
import matplotlib.pyplot as plt
data = root_open("./391.0_data_kmu2.root");
data_plot = data.Get("cut_211111_MUVEfficiency")
rplt.hist(data_plot,  color="blue")
#rplt.errorbar(kmu3_data_plot, yerr=False, label="data", emptybins=False, fmt='r-')
rplt.errorbar(data_plot,label='bla', fmt='ro')

plt.savefig("./test.pdf")
````
This output is created:
![test](https://cloud.githubusercontent.com/assets/6237108/6315219/24c9bf48-b9fb-11e4-8c7c-93f161a3c00c.png)

As you see all my color commands are ignored. Do I miss something?

**Comment by srstevenson** [comment]
`root2matplotlib.hist` [doesn't pass](https://github.com/rootpy/rootpy/blob/b6331ccb0b5140fac9f8fbb11acb6db289ab7a00/rootpy/plotting/root2matplotlib.py#L284-L285) `color` to [`root2matplotlib.step`](https://github.com/rootpy/rootpy/blob/b6331ccb0b5140fac9f8fbb11acb6db289ab7a00/rootpy/plotting/root2matplotlib.py#L596-L634) (I think this is a bug), so the line colour is [taken from the ROOT histogram](https://github.com/rootpy/rootpy/blob/b6331ccb0b5140fac9f8fbb11acb6db289ab7a00/rootpy/plotting/root2matplotlib.py#L629-L630).  As a workaround you can set the colour before plotting:
```python
data_plot = data.Get("cut_211111_MUVEfficiency")
data_plot.color = "blue"
rplt.hist(data_plot)
```
You can use any colour understood by either ROOT or matplotlib.

**Comment by oozaru-hunter** [comment]
Thank you, with 

    data_plot.color = "blue"

I can indeed change the line color.
![test](https://cloud.githubusercontent.com/assets/6237108/6315950/02c6f7ea-ba13-11e4-826d-de627f497257.png)

However, I seems I cannot change the fill color. At least neither
     data_plot.SetFillColor("blue") # nor
     data_plot.fillcolor = "blue"
has any effect.

**Comment by ndawe** [comment]
Fixed in #629

**Comment by srstevenson** [comment]
:+1:
